### PR TITLE
Bug 1190140 - [Calendar] Convert checkboxes to gaia-checkbox component

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1190140

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=47eac101b30909d755ca429d58ef040b5d5733b9

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=06d4b8ac4bf6d499d673b98306635b7b5af3a968

File: apps/calendar/style/settings.css [hunk 1]
```diff
@@ -99,18 +99,15 @@ not actually shown since the top header should be transparent.
...
   text-overflow: ellipsis;
   width: -moz-calc(100% - 2rem);
   height: 100%;
-}
-
-#settings .calendars label > span {
-  display: block;
-  float: left;
+  /* !important needed to override component scoped style to match legacy styles */
+  font-size: 1.7rem !important;
```
Comment (millermedeiros) [inline apps/calendar/style/settings.css]: also need to set the `color: #606060;` otherwise text will be darker than before.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a31f5cab49c83c3c46e7c365080d83c6cca21b4f

**Pull request merged**